code/controllers/subsystem/reagents.dm

```diff
@@ -0,0 +1,49 @@
+//Used for active reactions in reagents/equilibrium datums
+
+SUBSYSTEM_DEF(reagents)
+	name = "Reagents"
+	init_order = INIT_ORDER_REAGENTS
+	priority = FIRE_PRIORITY_REAGENTS
+	wait = 0.5 SECONDS //You might think that rate_up_lim has to be set to half, but since everything is normalised around delta_time, it automatically adjusts it to be per second. Magic!
+	flags = SS_KEEP_TIMING
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
+
+	var/list/processing = list()
+	var/list/currentrun = list()
+	///What time was it when we last ticked
+	var/previous_world_time = 0
+
+/datum/controller/subsystem/reagents/Initialize()
+	//So our first step isn't insane
+	previous_world_time = world.time
+	//Build GLOB lists - see holder.dm
+	build_chemical_reagent_list()
+	build_chemical_reactions_list()
+	return ..()
+
+//Comment to delete: I don't really understand SS that well. Please review me
+/datum/controller/subsystem/reagents/stat_entry(msg)
+	msg = "reagents:[length(processing)]"
```

Comment:
This string ends up being an entry in a line, it'd be better to keep it in the old format of P:[length(processing)] as it keeps things readable. Also you should again most likely take this behavior from the processing subsystem

Reply:
I'll change it to `P:[length(processing)]` - I think it's verbatim what processing has.